compiler.go

```diff
@@ -955,8 +960,10 @@ func (c *Compiler) checkCyclicImports(
 
 func (c *Compiler) compileModule(
 	node parser.Node,
-	moduleName, modulePath string,
+	moduleName string,
+	modulePath string,
```

Comment:
looks like we're always passing the same value for `moduleName` and `modulePath`. Let's make it a single parameter `modulePath`.

Reply:
Ok